packages/main/src/themes/sap_horizon/Token-parameters.css

```diff
@@ -6,10 +6,18 @@
 	--_ui5_token_selected_focus_outline: none;
 	--_ui5_token_focus_outline: none;
 	--_ui5_token_outline_offset: 0.125rem;
-	--_ui5_token_selected_hover_background: var(--sapButton_Selected_Background);
+	--_ui5_token_selected_hover_background: var(--sapButton_Selected_Hover_Background);
 	--_ui5_token_selected_hover_border_color: var(--sapButton_Selected_BorderColor);
 	--ui5_token_focus_pseudo_element_content: '';
-	--_ui5_token_focus_outline_border_radius: 0.625rem;
+	--_ui5_token_icon_padding: 0.25rem 0.5rem;
+	--_ui5_token_border_radius: 0.375rem;
+	--_ui5_token_focus_outline_border_radius: .5rem;
+	--_ui5_token_text_color: var(--sapTextColor);
+	--_ui5_token_hover_background: var(--sapButton_Hover_Background);
+	--_ui5_token_selected_box_shadow: var(--sapContent_Selected_Shadow);
+	--_ui5_token_selected_text_font_family: var(--sapFontSemiboldDuplexFamily);
+	--_ui5_token_selected_internal_border_bottom: var(--sapButton_BorderWidth) solid var(--sapButton_Selected_BorderColor);
```

Comment:
--_ui5_token_selected_internal_border_bottom: var(--sapButton_BorderWidth) -> is it correct to use the --sapButton_BorderWidth parameter, as it is not mentioned in the specification but "2px/0.125rem". I see that the value is the same but if parameters value is changed in future, that border will be also affected. The same is valid for Evening Horizon